src/Microsoft.Extensions.Logging.Configuration/baseline.netcore.json

```diff
@@ -1,5 +1,5 @@
 {
-  "AssemblyIdentity": "Microsoft.Extensions.Logging.Configuration, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
The fact that this is listed as having been generated off of 2.1.0 seems concerning.